### DISCUSSION TITLE
Duration predictor output format

### DISCUSSION THREAD

**Discussion by WorkingJack** [p0p4k/vits2_pytorch#86] [Q&A] [answered]
Hello, if I didn't get it wrong, the duration predictor outputs the representation value of durations, I'm wondering is it possible to convert it into real time unit like seconds? Any suggestion or idea is appreciated, thank you!

**Comment by p0p4k** [comment, marked as answer]
Frame number x hop length / sample rate

**Reply by WorkingJack** [reply]
Well, I didn't expect it to be that easy, thank you!!

**Reply by p0p4k** [reply]
Do some reading on TTS papers so you can quickly get hold of these basic things. Good luck! Ask me if you have more questions.
